eve/main.yml

```diff
@@ -23,6 +23,18 @@ stages:
       images:
           docker-builder: eve/workers/pod-builder
     steps:
+    - ShellCommand:
+        name: Wait for Docker daemon to be ready
+        command: |
+          bash -c '
+          for i in {1..10}
+          do
+            docker --version &> /dev/null && exit
```

Comment:
`&& exit 0` ?

Reply:
A shell expert (@alexandre-allard-scality) told me that `exit` takes by default the return code of the latest command, `$?`. Since you have `some-cmd && exit`, you know for sure that `$? = 0`.